rpm/0003-Use-simple-engine-by-default.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
+Date: Mon, 8 Jan 2024 21:19:04 +0200
+Subject: [PATCH] Use simple engine by default
+
+---
+ data/rygel.conf | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/data/rygel.conf b/data/rygel.conf
+index 807f3a86..2a242f03 100644
+--- a/data/rygel.conf
++++ b/data/rygel.conf
+@@ -23,7 +23,7 @@ picture-upload-folder=@PICTURES@
+ 
+ # Default media engine to load. If not specified, the engine directory is
+ # searched recursively and the first engine found is loaded.
+-media-engine=librygel-media-engine-gst.so
```

Comment:
In the commit history it looks like gstreamer integration was disabled many years ago when it required gst 1.0 and we were still on 0.10. Might be worth trying it again now instead of doing this.

Reply:
The gstreamer plugin needs an extra dependencies which we don't have packaged at the moment. Might be something to look in to later, main benefit from gstreamer plugin would be transcoding support.